packages/neon/neon_dashboard/test/set_weather_location_dialog_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:neon_dashboard/l10n/localizations.dart';
+import 'package:neon_dashboard/src/widgets/set_weather_location_dialog.dart';
+import 'package:neon_framework/l10n/localizations.dart';
+import 'package:neon_framework/theme.dart';
+
+void main() {
+  testWidgets('Set weather location dialog', (tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        localizationsDelegates: const [
+          ...NeonLocalizations.localizationsDelegates,
+          ...DashboardLocalizations.localizationsDelegates,
+        ],
+        supportedLocales: const [
+          ...NeonLocalizations.supportedLocales,
+          ...DashboardLocalizations.supportedLocales,
+        ],
+        theme: ThemeData(
+          extensions: const [
+            NeonTheme(
+              branding: Branding(
+                name: '',
+                logo: SizedBox.shrink(),
+              ),
+            ),
+          ],
+        ),
+        home: const SizedBox.shrink(),
+      ),
```

Comment:
Let's wait for https://github.com/nextcloud/neon/pull/1547 so we can use the `TestApp`.